sklearn/datasets/_svmlight_format.pyx

```diff
@@ -4,6 +4,8 @@
 #          Lars Buitinck
 #          Olivier Grisel <olivier.grisel@ensta.org>
 # License: BSD 3 clause
+#
+# cython: language_level=3str, boundscheck=False, wraparound=False
```

Comment:
typo?

Reply:
if you mean "3str" not that's intentional,

> The 3str option enables Python 3 semantics but does not change the str type and unprefixed string literals to unicode when the compiled code runs in Python 2.x

because that uses C `strchr` and this was failing otherwise. But yeah, ideally I should investigate how to work around that and make it work with `language_level=3`.